src/app/qgsattributetypedialog.cpp

```diff
@@ -93,7 +89,11 @@ QgsAttributeTypeDialog::QgsAttributeTypeDialog( QgsVectorLayer *vl, int fieldIdx
   restoreGeometry( settings.value( QStringLiteral( "Windows/QgsAttributeTypeDialog/geometry" ) ).toByteArray() );
 
   constraintExpressionWidget->setLayer( vl );
+
+  // TODO, what should this help be connected to?
```

Comment:
I'd say: to somewhere in this section http://docs.qgis.org/testing/en/docs/user_manual/working_with_vector/vector_properties.html#fields-properties. Isn't it [already implemented ](https://github.com/qgis/QGIS/pull/5467/files#diff-be501c6a669cec8cc989f027704b1e2eR423) (unless I misunderstood - which is really probable)?

Reply:
Solved it now like this, that if you are on this tab (Form) this help-site is loaded when pressing help button of vectorlayerproperties dialog. Because it does not make sence to have two help-buttons.